content/docs/concepts/security/index.md

```diff
@@ -771,6 +770,51 @@ spec:
     name: "products-viewer"
 {{< /text >}}
 
+### Using Istio authorization on plain TCP protocols
+
+The above examples showed a typical way to use Istio authorization on services using HTTP protocol.
+In that case, all fields in a `ServiceRole` and `ServiceRoleBinding` are supported.
+
+Istio authorization also supports services using any plain TCP protocols, such as mongodb. In this case,
+you specify the `ServiceRole` and `ServiceRoleBinding` the same way as for a HTTP service, except that
+you cannot use the fields that are only applicable to HTTP services, which includes `paths` and `methods`
+in `ServiceRole`, and `group` in `ServiceRoleBinding`.
+
+For the supported constraints and properties that could be used for TCP service, please refer to the
+[constraints and properties page](/docs/reference/config/authorization/constraints-and-properties/).
+
+If any HTTP only fields are used for a TCP service, the `ServiceRole` or `ServiceRoleBinding` custom
+resource will be ignored completely as if there is no such policy.
+
+Assume you have a mongodb service serving on port 27017, the following example shows how to only
```

Comment:
Clarity and specificity: Please be clear by adding the proper nouns after the names of the objects. If at any point you are unsure of what noun to use, imagine how hard it would be for someone just learning the project to know it. To fix this text, I suggest:
Assuming you have a MongoDB service on port 27017, the following example configures a service role and a service role binding to only allow the `bookinfo-ratings-v2` service in the Istio mesh to access the MongoDB service.